wagtail/admin/ui/tables/__init__.py

```diff
@@ -299,6 +300,29 @@ def __init__(self, **kwargs):
         )
 
 
+class LocaleColumn(StatusTagColumn):
```

Comment:
random thought: it doesn't look like we have test for most of the columns defined here. There are indirect ones throught various viewset/listing tests. Worth creating an issue?

Reply:
Not sure. Maybe when we publicly document them and the tables framework in general, but for now I don't see the urgency as we mostly already have the integration tests.